src/main/java/racingcar/View/OutputView.java

```diff
@@ -0,0 +1,23 @@
+package racingcar.View;
+
+import static racingcar.View.ViewConstants.NAME_DELIMITER;
+import static racingcar.View.ViewConstants.RACE_RESULT;
+import static racingcar.View.ViewConstants.WHITE_SPACE;
+import static racingcar.View.ViewConstants.WINNER_IS;
+
+import java.util.List;
+
+public class OutputView {
+    public void printResultPhrase() {
+        System.out.println(System.lineSeparator() + RACE_RESULT);
+    }
+
+    public void printRaceProgress(String roundResult) {
+        System.out.println(roundResult);
+    }
+
+    public void printWinners(List<String> winnerNames) {
+        String winners = String.join(NAME_DELIMITER + WHITE_SPACE, winnerNames);
+        System.out.printf("%s%s", WINNER_IS, winners);
```

Comment:
중요한건 아니지만
```suggestion
        System.out.printf(WINNER_IS + winners);
```
와 같이 작성하는 방법도 있습니다.